fastestimator/estimator.py

```diff
@@ -56,24 +56,46 @@ def __init__(self,
                  monitor_names: Union[None, str, Iterable[str]] = None):
         self.pipeline = pipeline
         self.network = network
+        self.epochs = epochs
```

Comment:
this should not be redundantly recorded, self.system is source of truth for this info

Reply:
done